external-secret-operator/add-external-secret-prefix/add-external-secret-prefix.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: add-external-secret-prefix
+  annotations:
+    policies.kyverno.io/title: Add prefix to exernal secret
+    policies.kyverno.io/category: ExternalSecretOperator
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: ExternalSecret
+    kyverno.io/kyverno-version: 1.7.1
+    policies.kyverno.io/minversion: 1.6.0
+    kyverno.io/kubernetes-version: "1.23"
+    policies.kyverno.io/description: >- 
+      This Policy mutates secretRef key to add a prefix.
+      External Secret Operator proposes to use kyverno to force ExternalSecrets
+      to have namespace prefix so that kubernetes administrators do not need to
+      define permissions and users per namespace.
+      Doing this developers are abstracted by administrators naming convenction and will not 
+      be able to access to secrets from other namespaces.
+      In this example on patch change "prefix-" for your prefix. For example: {{ request.namespace }}
+spec:
+  rules:
+  - name: add-external-secret-prefix
+    match:
+      resources:
```

Comment:
Can you please nest this under either an `any` or `all` block, please? We're standardizing on that in the CRD.